lasagne/layers/conv.py

```diff
@@ -933,6 +935,207 @@ def convolve(self, input, **kwargs):
 Deconv2DLayer = TransposedConv2DLayer
 
 
+class TransposedConv3DLayer(BaseConvLayer):
```

Comment:
That's also the reason why I had two commits in #791. Makes it easy to undo the compatibility changes when we drop Theano 0.8 compatibility.

Hmm, in the end I would like to have two commits for this PR as well -- are you okay with me rearranging and squashing the commits as needed? I can add the missing `# pragma: no cover` as well then. (You'll still be the author of all commits.)